--- a/debian/control
+++ b/debian/control
@@ -8,7 +8,11 @@
                dh-exec,
                dh-golang,
                golang-any,
-               golang-github-aws-aws-sdk-go-dev,
+               golang-github-aws-aws-sdk-go-v2-dev,
+               golang-github-azure-azure-sdk-for-go-dev,
+               golang-github-containers-libhvee-dev,
+               golang-github-mdlayher-vsock-dev,
+               golang-github-opencontainers-selinux-dev, 
                golang-github-beevik-etree-dev,
                golang-github-coreos-go-json-dev,
                golang-github-coreos-go-systemd-dev,
