diff --git a/dracut/30ignition/module-setup.sh b/dracut/30ignition/module-setup.sh
index 86f24721..df599e8c 100755
--- a/dracut/30ignition/module-setup.sh
+++ b/dracut/30ignition/module-setup.sh
@@ -130,6 +130,6 @@ installkernel() {
 
     # required by nvidiabluefield platform to read ignition file through bootfifo sysfs interface
     if [[ ${DRACUT_ARCH:-$(uname -m)} == aarch64 ]]; then
-        instmods -c mlxbf_bootctl
+        instmods mlxbf_bootctl
     fi
 }
